firefox_puppeteer/tests/test_software_update.py

```diff
@@ -0,0 +1,108 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+from firefox_puppeteer.api.software_update import SoftwareUpdate
+
+
+class TestSoftwareUpdate(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.software_update = SoftwareUpdate(self.get_marionette)
+        self.saved_mar_channels = self.software_update.mar_channels.channels
+        self.software_update.mar_channels.channels = 'expected,channels'
```

Comment:
> I included it because |build_info| includes the MARChannels, so I wanted
> to be able to set an expected value for that in order to do the assert.
> I can remove it and just use |assertTrue| if you would prefer that.

I see. That makes sense then. So lets leave it as is. Thanks.